src/devices/tuya.ts

```diff
@@ -13124,6 +13124,25 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE284_nhgdf6qr']),
```

Comment:
Since the definition is the same, could you add this to the `TS0601_soil` definition and use `tuya.whitelabel` for correct detections (search in this file for some examples)